### DISCUSSION TITLE
Wrong entities by Fibaro rollershutter modules

### DISCUSSION THREAD

**Discussion by jansendejong** [zwave-js/zwave-js#4023] [Q&A]
Hi all, 

Last week I went from OZW to Zwave js. 
My 2 cheapy TKB TZ75 modules have the exact same entities like they had in OZW. 
Butt the Fibaro FGR223 modules just have 2 entities that doesn't do anything. 

![Screenshot_20220103-180729_Home Assistant.jpg](https://user-images.githubusercontent.com/31671096/147969018-8c61372f-0927-4ae7-83b4-04705a77aa22.jpg)

![Screenshot_20220103-182020_Home Assistant.jpg](https://user-images.githubusercontent.com/31671096/147969022-005b7160-1553-4f55-b3d6-f04799072c24.jpg)

How can I solve this?

**Comment by AlCalzone** [comment]
Please start by reporting this in https://github.com/home-assistant/core, so the HA devs can figure out if this is an issue in HA or here.
